runtime/src/main/java/io/novafoundation/nova/runtime/multiNetwork/chain/model/Chain.kt

```diff
@@ -46,6 +46,17 @@ data class Chain(
 
     val assetsById = assets.associateBy(Asset::id)
 
+    data class Icon(
+        val name: String,
+        val baseUrl: String,
+    ) {
+        companion object;
+
+        constructor(url: String) : this("", url)
+
+        fun getIconUrl(): String = "$baseUrl/$name"
+    }
```

Comment:
I understand that this is the easy  and quick way do achieve that. But it is not the right one.`ChainRegistry` and `Chain` already has too much responsibility and we definietely dont want to bring user prefererences in here.  We should have a separate entity, like `IconAppearanceProvider` that will construct icon urls based on chain and (internally) the mode and apply this change everywhere the icon is used. Thats what I meant under "it will require changes in a lot of places"